src/shapes/Text/Text.ts

```diff
@@ -40,6 +44,8 @@ import {
   JUSTIFY_RIGHT,
 } from './constants';
 import { CENTER, LEFT, RIGHT, TOP, BOTTOM } from '../../constants';
+import { isFiller } from '../../util/typeAssertions';
+import type { Gradient, Pattern } from 'fabric/node';
```

Comment:
Hello

Reply:
I hate this
I thought of adding a step to publishing (altering `package.json`) so this won't happen to us.

Reply:
oopppsss i ll fix it.
This is the stupid VSCODE.
The automatic import MICROSOFT NEVER WORK things